src/fontra/views/editor/editor.js

```diff
@@ -1017,25 +1017,25 @@ export class EditorController {
     let pastedGlyph;
 
     const plainText = await readFromClipboard("text/plain");
-    if (plainText) {
-      pastedGlyph = await this.fontController.parseClipboard(plainText);
+    if (!plainText) {
+      return;
     }
 
-    const isTheSameGlyphPath =
-      JSON.stringify(pastedGlyph) === localStorage.getItem("clipboardSelection.glyph");
+    const isClipboardValid =
```

Comment:
I prefer to name it more explicitly, perhaps even`isOurLocalClipboardValid`. It is not about the clipboard in general after all.

Reply:
Just to clarify: by 'Local Clipboard' you refer to the localStorage values, right?